content/objects/proc/alert.md

```diff
@@ -16,4 +16,8 @@ name = "Button1" # AUTOGEN STATIC
 name = "Button2" # AUTOGEN STATIC
 [[extra.args]]
 name = "Button3" # AUTOGEN STATIC
-+++
\ No newline at end of file
++++
+
+Displays a customizable alert message to a user (either a {{ mob() }} or {{ client() }}). It accepts a title, message, and up to three buttons as text. If no buttons are specified, the first defaults to "Ok."
+
+If no user is explicitly provided, the current user is used and arguments are shifted. The function checks for a valid connection; if none exists, it returns "OK." Otherwise, it sends the alert and returns the user's interaction.
```

Comment:
```suggestion
If no user is explicitly provided, the current user is used and arguments are shifted. The function checks for a valid connection; if none exists, it returns "OK" Otherwise, it sends the alert and returns the user's interaction.
```